modules/ROOT/pages/mongodb/mongodb-connector-6-0-design-center.adoc

```diff
@@ -0,0 +1,78 @@
+= MongoDB Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Design Center:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Design Center field values, see the xref:mongodb/mongodb-connector-reference.adoc[MongoDB Connector Reference].
+
+== Configure the Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click the name of the trigger card.
+. To use the MongoDB connector as an input source trigger, choose the *Object Listener* input source.
++
+Alternatively, you can trigger your app using HTTP Listener or Scheduler.
++
+Design Center automatically saves changes you make in a session.
+
+== Configure the Target Component
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector name.
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource.
+. Click *Test Connection* to ensure you have access to MongoDB.
+
+To set up a MongoDB configuration use the following fields:
+
+[%header%autowidth.spread]
```

Comment:
this table is old. Some of those fields are not available anymore. Please your help here @dariohvega p

Reply:
The new table must be: 

```
[%header%autowidth.spread]
|===
|Fields |Description
|Required Libraries |Reference to the MongoDB Driver. The version must be 3.11 or higher.
|Servers |List of server addresses to use for the connection.
|Database |MongoDB database
|Username |MongoDB username
|Password |MongoDB password
|===
```